Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM keydonix/parity-instantseal
+
+# Lifted from: https://github.com/nodejs/docker-node/blob/master/8/Dockerfile
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+ENV NODE_VERSION=8.11.2 \
+  YARN_VERSION=1.7.0 \
+  ARCH=x64
+
+COPY node-v$NODE_VERSION-linux-$ARCH.tar.xz .
+RUN tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
+  && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
+  && ln -s /usr/local/bin/node /usr/local/bin/nodejs
+
+
+COPY yarn-v$YARN_VERSION.tar.gz .
+RUN mkdir -p /opt \
+  && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/ \
+  && rm yarn-v$YARN_VERSION.tar.gz \
+  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarn /usr/local/bin/yarn \
+  && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg
```

Comment:
Out of curiosity, why yarn instead of NPM?  Doesn't NPM come with node already?

Reply:
Just cargo-cult'ing the node8 Dockerfile. I have run into a few places where yarn was required and npm was specifically not working, so I figure that's why? npm is included in image as well